### ISSUE TITLE
Entity Dialog sometimes loads without Entity ID

### ISSUE THREAD

**Issue by ReatKay** [volkanceylan/Serenity#4263]
Very randomly, an Entity Dialog loads in isNew() state, when opened as Entity Edit. The Entity behind is loaded. It turned out, that the Entity Id is missing - not just on the dialog, but on the whole Entity under the dialog. But funny that all the other columns are properly loaded. Also the location.href is in new mode, even the link used to open is clearly showing id & edit. Somehow, doesnt make any sense to me. How can the entity be loaded, but not the primary key -laugh-

Did anybody else has been confronted with that behaviour?

Serenity .NET , 
Serenity NuGet Packages v. 3.8.1. I tried to solve it by updating to 3.8.6, but 2 packages lead to for me unresolvable errors (due to my lack in knowledge around Serenity/ASP.NET MVC yet) - and I didn't want to to use packages in different versions.

**Comment by Zahar661** [comment]
You have all this?
![image](https://user-images.githubusercontent.com/26430136/52688141-870a0680-2f6e-11e9-8f1f-f7525097bbab.png)
![image](https://user-images.githubusercontent.com/26430136/52688175-ba4c9580-2f6e-11e9-86b3-8dcbb3207bfc.png)

**Comment by ReatKay** [comment]
I found the issue. I used for one dialog a sample from the web. And as my dialogs share the first few rows, I usually just copy that. Now I saw on the sample, there have been some "extra" attributes on the Id.. and this caused unpredictable behaviour.

Should have seen it before. PrimaryKey + Unique + Identity + AutoIncrement. Wtf ^^ - i think a bit too much. Removed the overdone stuff , working again.

Can be closed -> solved
